WordPress/Classes/ViewRelated/Me/MeHeaderView.m

```diff
@@ -243,4 +248,44 @@ - (IBAction)handleHeaderPress:(UIGestureRecognizer *)sender
     }
 }
 
+#pragma mark - Drop Interaction Handler
+- (BOOL)dropInteraction:(UIDropInteraction *)interaction
+       canHandleSession:(id<UIDropSession>)session API_AVAILABLE(ios(11.0))
+{
+    BOOL isAnImage = [session canLoadObjectsOfClass:[UIImage self]];
+    BOOL isSingleImage = [session.items count] == 1;
+    return (isAnImage && isSingleImage);
+}
+
+- (UIDropProposal *)dropInteraction:(UIDropInteraction *)interaction
+                   sessionDidUpdate:(id<UIDropSession>)session API_AVAILABLE(ios(11.0))
+{
+    CGPoint dropLocation = [session locationInView:self];
+    UIDropOperation dropOperation;
+    
+    if (CGRectContainsPoint(self.gravatarImageView.frame, dropLocation)) {
+        dropOperation = UIDropOperationCopy;
+    } else {
+        dropOperation = UIDropOperationCancel;
+    }
+    
+    UIDropProposal *dropProposal = [[UIDropProposal alloc] initWithDropOperation:dropOperation];
+    
+    return  dropProposal;
+}
+
+- (void)dropInteraction:(UIDropInteraction *)interaction
+            performDrop:(id<UIDropSession>)session API_AVAILABLE(ios(11.0))
+{
+    [self.gravatarImageView depressSpringAnimation:nil];
```

Comment:
Nice, using the existing animation 👍 In fact, I wonder if we could do this in the drop delegate's `sessionDidEnter` and `didExit` methods (I think those are the right ones), to depress the image view when you're hovering the item over it. What do you think?